src/linking/remove_internal_symbols.cpp

```diff
@@ -144,6 +144,16 @@ void remove_internal_symbols(
     if(is_type)
     {
       // never EXPORTED by itself
+        // Without this code the symbol is lost when compiling with
```

Comment:
Here is where I _was_ copying the ID_typedef into the concrete type (without this [or the above change]) then we don't have the typedef'd name of the type. However, as described in the comment, this won't work with chained or multiple typedefs.